src/Polyfill/RuntimeHelpers.cs

```diff
@@ -0,0 +1,31 @@
+// <auto-generated />
+
+#if (NET46X && FeatureValueTuple) || NET47X || NET48X || NETSTANDARD2_0 || NETCOREAPP2X
+
+namespace System.Runtime.CompilerServices;
+
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using Link = System.ComponentModel.DescriptionAttribute;
+
+/// <summary>
+/// Provides a set of static methods and properties that provide support for compilers.
+/// This class cannot be inherited.
+/// </summary>
+[ExcludeFromCodeCoverage]
+[DebuggerNonUserCode]
+#if PolyPublic
+public
+#endif
+static partial class RuntimeHelpers
+{
+    /// <summary>
+    /// Provides a fast way to initialize an array from data that is stored in a module.
+    /// </summary>
+    /// <param name="array">The array to be initialized.</param>
+    /// <param name="fldHandle">A field handle that specifies the location of the data used to initialize the array</param>
+    [Link("https://learn.microsoft.com/en-us/dotnet/api/system.runtime.compilerservices.runtimehelpers.initializearray")]
+    [Intrinsic]
+    public static extern void InitializeArray(Array array, RuntimeFieldHandle fldHandle);
```

Comment:
@SimonCropp, after adding this method, everything compiles and tests are green. But I have a few questions:

1. This method should already exist in all the TFMs. Can it cause runtime errors?
![image](https://github.com/SimonCropp/Polyfill/assets/4518483/5bbfe3e3-ce48-4bb5-9ff1-c238ccedbfbb)

2. While all tests run fine, the ReSharper runner shows the following errors. Any idea how to fix them? Is it safe to ignore?
![image](https://github.com/SimonCropp/Polyfill/assets/4518483/a05e0c41-d4a2-47c1-87ff-c6139024f4ca)

3. I created 2 partial `RuntimeHelpers` classes, as `InitializeArray` is not necessarily related to the `IndexRange` feature and theoretically can be required by other features added in the future. Is it ok, or do you prefer a single class?

Reply:
try this
```
// <auto-generated />

#if (NET46X && FeatureValueTuple) || NET47X || NET48X || NETSTANDARD2_0 || NETCOREAPP2X

namespace System.Runtime.CompilerServices;

using System.Reflection;
using System.Diagnostics;
using System.Diagnostics.CodeAnalysis;
using Link = System.ComponentModel.DescriptionAttribute;

/// <summary>
/// Provides a set of static methods and properties that provide support for compilers.
/// This class cannot be inherited.
/// </summary>
[ExcludeFromCodeCoverage]
[DebuggerNonUserCode]
#if PolyPublic
public
#endif
static partial class RuntimeHelpers
{
    static readonly Action<Array,RuntimeFieldHandle> initializeArrayAction;

    static RuntimeHelpers()
    {
        //CompilationRelaxations is co located with RuntimeHelpers
        var runtimeHelpersType = typeof(CompilationRelaxations)
            .Assembly
            .GetType("System.Runtime.CompilerServices.RuntimeHelpers");
        var initializeArrayMethod = runtimeHelpersType.GetMethod(
            "InitializeArray",
            BindingFlags.Static | BindingFlags.Public,
            null,
            [typeof(Array), typeof(RuntimeFieldHandle)],
            null)!;
        initializeArrayAction = (Action<Array,RuntimeFieldHandle>)initializeArrayMethod.CreateDelegate(typeof(Action<Array,RuntimeFieldHandle>));
    }

    /// <summary>
    /// Provides a fast way to initialize an array from data that is stored in a module.
    /// </summary>
    /// <param name="array">The array to be initialized.</param>
    /// <param name="fldHandle">A field handle that specifies the location of the data used to initialize the array</param>
    [Link("https://learn.microsoft.com/en-us/dotnet/api/system.runtime.compilerservices.runtimehelpers.initializearray")]
    public static void InitializeArray(Array array, RuntimeFieldHandle fldHandle) =>
        initializeArrayAction(array, fldHandle);
}
#endif
```

and then we dont need IntrinsicAttribute